### ISSUE TITLE
trace 12 ~ trace 14 관련 문의드립니다.

### ISSUE THREAD

**Issue by ghost** [SNU-ARC/2024_spring_sysprog_Lab4#26]
조교님, 안녕하십니까?
다름 아니라 trace 12 ~ trace 14 관련 궁금한 점이 있어서 질문드립니다.

trace 14(trace 12, trace 13도 비슷합니다.)에서 주어진 reference/csapsh를 제 컴퓨터에서 실행한 결과는 아래와 같았으며, 

![rtest14](https://github.com/SNU-ARC/2024_spring_sysprog_Lab4/assets/170851753/941ba132-2a5c-4d40-baf0-fb214ff0b4e2)

실제 주어진 out.14는 아래와 같았습니다.

![rtest14-1](https://github.com/SNU-ARC/2024_spring_sysprog_Lab4/assets/170851753/d85814d6-af37-4259-9b5e-088dd5c9fa63)

혹시 ps command에서 pid 뿐만 아니라 process도 약간 다른데 이를 반영해서 채점에서 이루어지는지 궁금합니다.

**Comment by Jong25** [comment]
네, 반영하여 채점할 계획이니 걱정하지 않으셔도 됩니다. (csapsh에서 실행한 process들이 들어있는지 정도 확인해보시기 바랍니다)

**Comment by m-joon-ixix** [comment]
같은 질문을 했었어서 참고하시라고 링크 걸어드립니다. #22

**Comment by ghost** [comment]
감사합니다, 조교님
